app/src/main/java/com/example/pehchaan/activity/AddContactsActivity.java

```diff
@@ -35,17 +35,12 @@ public void onClick(View view) {
             public void onClick(View view) {
                 if (binding.firstNameET.getText().toString().isEmpty()) {
                     binding.firstNameET.setError("* Required");
-                } else if (binding.lastNameET.getText().toString().isEmpty()) {
-                    binding.lastNameET.setError("* Required");
-                } else if (binding.lastNameET.getText().toString().isEmpty()) {
-                    binding.lastNameET.setError("* Required");
-                } else if (binding.phoneNumET.getText().toString().isEmpty()) {
+                }else if (binding.phoneNumET.getText().toString().isEmpty()) {
                     binding.phoneNumET.setError("* Required");
-                } else if (binding.phoneNumET.getText().toString().length() < 10) {
```

Comment:
Every mobile no should consist of 10 digits. So this seems to be a valid condition. Why did u remove this?

Reply:
Because I have added this line in the EditView of the AddContactsActivity
=>   *android:maxLength="10"*

This line is sufficient for this 10-digit mobile number.

Reply:
It doesn't restrict the user to a lower limit, maxLength restricts the user to an upper limit.

Reply:
On complication, user would be able to add their contacts with phone number digits below 10 too. This shouldn't be happening.